### ISSUE TITLE
What to do with derived variables?

### ISSUE THREAD

**Issue by FlukeAndFeather** [FlukeAndFeather/cegR#10]
The following environmental variables are listed as derived:

<google-sheets-html-origin>

Derived Variables (rasters) | Parent | Time scale | Parent provider
-- | -- | -- | --
EKE | Derived from Altimetry above: ugosa and vgosa | Historical | AVISO via CMEMS
EKE | Derived from Altimetry above: ugosa and vgosa | NRT | AVISO via CMEMS
sla_sd | Derived from Altimetry above: sla | Historical | AVISO via CMEMS
sla_sd | Derived from Altimetry above: sla | NRT | AVISO via CMEMS
sst_sd | Derived from SST above: | Historical | GHRSST via CMEMS
sst_sd | Derived from SST above: | NRT | GHRSST via CMEMS
adt_sd | Derived from Altimetry above: adt | Historical | AVISO via CMEMS
adt_sd | Derived from Altimetry above: adt | NRT | AVISO via CMEMS
Rugosity | bathy_sd; derived from Bathymetry above: bathy | NA | ETOPO180

</google-sheets-html-origin>

What should we do about these in the new workflow?

@heatherwelch

**Comment by FlukeAndFeather** [comment]
To clarify, the pasted table comes from the [Global_environmental_variables](https://docs.google.com/spreadsheets/d/1tkCH9C0dQd2U3gwz17KtvCH1NoExn4n8ReBf5pXlxVk/edit?usp=drive_link) workbook. We should figure out how to represent those variables in \[[CEG Geospatial Data Sources](https://docs.google.com/spreadsheets/d/15duOwuf5PCRfLEXWJzopdzQbZkjHhTvHbop2xDWWcME/edit#gid=0)\]Variables!B55:E63.

**Comment by HeatherWelch** [comment]
Thanks Max. I don't think we actually need to do anything with these which is nice. All of the sds (adt, sst, sla, rugosity) will be calculable with your roving window arguments we talked about adding to the function on our last call. EKE is derived from ugosa and vgosa, and the end user can calculate the rest of the equation on their own after extracted those two variables. Does that seem correct to you?

One thing we've discussed before but I forgot to double check in our last call - the end user will be able to extract at the native resolution ie. no focal window right? Flagging to add this into the documentation - maybe its just leave window radius and statistic (e.g. mean) blank?
